web/modules/custom/dpl_related_content/dpl_related_content_tests/dpl_related_content_tests.info.yml

```diff
@@ -0,0 +1,30 @@
+name: DPL related content - test suite
+description: Testing the functionality of DPL Related Content.
+package: DPL
+type: module
+core_version_requirement: ^10
+dependencies:
+  - default_content:default_content
```

Comment:
>Should this be a hidden module to avoid having it show up in module lists?

I kind of want to avoid the `hidden` thing, because it really messed with me recently - It makes sense for deprecated stuff, but for this, i'd like to keep it visible.

>I wonder if we could achieve the same thing using a proper programmatic test using either Cypress or PHPUnit.

We deffo could - However, that is way beyond the scope of this task.
I'd love to talk to you some time about how to do this.